recipes/kit4dl/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "kit4dl" %}
+{% set version = "2023.08b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kit4dl-{{ version }}.tar.gz
+  sha256: 7ab8579afdf042b32559b9cc8eac72ed34d3dbd1fba87e0dfa09353cf7d65ce4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kit4dl = kit4dl.cli.app:run
+
+requirements:
+  host:
+    - python >=3.10
+    - pip    
+  run:
+    - python >=3.10
+    - pytorch
+    - lightning
+    - torchmetrics
+    - numpy >=1.23.0
```

The metadata says:

```suggestion
    - numpy <=1.25.0
```

not sure what is correct here though. I'd raise a question upstream.